packages/react/src/primitives/Link/__tests__/Link.test.tsx

```diff
@@ -0,0 +1,73 @@
+import { render, screen } from '@testing-library/react';
+import { kebabCase } from 'lodash';
+import { ComponentClassNames } from '../../shared';
+import { ComponentPropsToStylePropsMap } from '../../types';
+import { Link } from '../Link';
+import { Text } from '../../Text/Text';
+
+describe('Text: ', () => {
+  const linkText = 'My Link';
+
+  it('renders correct defaults', async () => {
+    render(<Link>{linkText}</Link>);
+
+    const link = await screen.findByText(linkText);
+    expect(link.innerHTML).toBe(linkText);
+    expect(link.nodeName).toBe('A');
+    expect(link.className).toContain(ComponentClassNames.Link);
```

Comment:
For testing `className`, it looks nicer to use the custom jest matchers [here](https://github.com/testing-library/jest-dom). It also provide many other handy matchers for us to use beyond testing against classname

`expect(link).toHaveClassName(ComponentClassNames.Link);`

Reply:
@zchenwei Can you write all my tests for me? You know all the best test methods :)